packages/dcos-integration-test/extra/api_session_fixture.py

```diff
@@ -2,13 +2,22 @@
 DcosApiSession object that will be injected into the pytest 'dcos_api_session' fixture
 via the make_session_fixture() method
 """
+from test_helpers import expanded_config
+
 from test_util.dcos_api_session import DcosApiSession, DcosUser, get_args_from_env
 from test_util.helpers import CI_CREDENTIALS
 
 
 def make_session_fixture():
     args = get_args_from_env()
-    args['auth_user'] = DcosUser(CI_CREDENTIALS)
-    dcos_api_session = DcosApiSession(**args)
+
+    exhibitor_admin_password = None
+    if expanded_config['exhibitor_admin_password_enabled'] == 'true':
+        exhibitor_admin_password = expanded_config['exhibitor_admin_password']
+
+    dcos_api_session = DcosApiSession(
+        auth_user=DcosUser(CI_CREDENTIALS),
```

Comment:
That's correct. I forgot about the pending tooling update: https://jira.mesosphere.com/browse/DCOS_OSS-1098